modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-monitoring-uis-and-apis.adoc
+
+:_content-type: PROCEDURE
+[id="accessing-third-party-monitoring-web-service-apis"]
+= Accessing third-party monitoring web service APIs
+
+[role="_abstract"]
+You can directly access third-party web service APIs from the command line for monitoring stack components such as Prometheus, Alertmanager, Thanos Ruler, and Thanos Querier. 
+
+The following example shows how to query the service API receivers for Alertmanager.
+This example requires that the associated user account be bound against the `monitoring-alertmanager-edit` role in the `openshift-monitoring` namespace and that the account has the privilege to view the route.
+This access only supports using a Bearer Token for authentication.
+
+[source, terminal]
+----
+$ host=$(oc -n openshift-monitoring get route alertmanager-main -ojsonpath={.spec.host})
+$ token=$(oc whoami -t)
+$ curl -H "Authorization: Bearer $token" -k "https://$host/api/v2/receivers"
+----
+
+To access Thanos Ruler and Thanos Querier service APIs, the requesting account must have `get` permission for the namespace to be queried.
```

Comment:
Move this to the prerequisites section?

Reply:
It's really more of an "additional information" sentence to note a difference with accessing these components versus what is being accessed in the example code. I've set it off with a Note admonition because of how it's functioning here.